VERSION

```diff
@@ -1 +1 @@
-yyyymmdd
+${yyyy}${mm}${dd}
```

Comment:
Not sure entirely how this works but do we want to have slashes (/) or dashes (-) for clarity of reading?

Reply:
That's a good observation. During the build process, a date like `20230914` might be produced. This approach to versioning artefacts, known as calendar versioning, is widely recognised. To enhance clarity and align it more closely with semantic versioning, the format `${yyyy}.${mm}.${dd}` is suggested, which would translate to `2023.09.14`. Both formats are supported by our template. It's up to individual engineering teams to decide which format suits their repositories best.

For our repository template, we'll adopt either the `${yyyy}.${mm}.${dd}` or the `x.y.z` version format. But this is yet to be decided. Thus, this file will undergo change again before producing the release candidate.

As part of this PR we wanted to introduce more intuitive and less error prone pattern `${...}`